client/src/index.js

```diff
@@ -8,19 +9,18 @@ import {
   store, persistor,
 } from './store/store';
```

Comment:
It makes sense to group all the store related folders.
E.x. you can have a src/store folder and inside it have index.js as a store main file + /actions and /reducers folders there too.